x-pack/legacy/plugins/snapshot_restore/public/app/components/policy_form/steps/step_settings.tsx

```diff
@@ -99,9 +105,7 @@ export const PolicyStepSettings: React.FunctionComponent<StepProps> = ({
           } else {
             updatePolicyConfig({
               indices:
-                selectIndicesMode === 'custom'
-                  ? indexPatterns.join(',')
-                  : [...(indicesSelection || [])],
+                selectIndicesMode === 'custom' ? [...indexPatterns] : [...(indicesSelection || [])],
```

Comment:
It seems like `indicesSelection` is always an Array, do we need the extra `|| []` ?